licode/contribute.html

```diff
@@ -101,6 +102,16 @@
 
         <p>The last step, once the design is accepted, code should be proposed via Pull Requests.</p>
       </section>
+      <section id="documentation">
+        <div class="page-header">
+          <h1>Documentation</h1>
+        </div>
+        <p>Documentation is one of the most important elements in an Open Source project. A good source code looses its value if developers cannot understand how it works! So we encourage you to also contribute here by fixing typos and improving the quality of our documents.</p>
+        <p>But it is also crucial to keep the documentation up to date with every code update. So when when you contribute a pull request that changes the APIs it is mandatory to also updates the corresponding documentation.</p>
+        <p>To make this things easier, our APIs documents are written using Markdown, a very ligth mark lenguage that uses plain text combined with some tags to add format and style. In Licode's documentation folder (<code>licode/doc/</code>) you will find the two main markdown files you would have to modify to introduce changes in the APIs, <code>client_api.md</code> and <code>server_api.md</code> for client and server APIs respectively.</p>
+        <p>To preview these documents while you are editing them you can use whatever markdown editor. However, if you want to see them as they are render in our website you have to serve them using <a href="http://www.mkdocs.org/" target="_blank">Mkdocs</a>. There you will find installation and usage instructions. Please, take into account you have to install version 15.0 for being compatible with Licode custom theme.</p>
```

Comment:
whatever -> any
render -> rendered
for being -> in order for it to be compatible